test/connector/tcp/mssql/start

```diff
@@ -1,27 +1,34 @@
 #!/bin/bash -ex
 
 mssql_host=mssql
-while getopts :dm:s: opt; do
+secretless_host=secretless
+mssql_edition=Developer # can also be Express, Standard, Enterprise, EnterpriseCore
+while getopts :dm:s:e: opt; do
     case $opt in
         d) dev_mode=true;;
         m) mssql_host=${OPTARG};;
         s) secretless_host=${OPTARG};;
+        e) mssql_edition=${OPTARG};;
        \?) echo "Unknown option -$OPTARG"; exit 1;;
     esac
 done
+
 # If the secretless host is not explicitly set on the command line,
```

Comment:
Nit: Comment should be updated since the `If the secretless host is not explicitly set on the command line` no longer applies.